package.json

```diff
@@ -9,6 +9,7 @@
     "lib"
   ],
   "scripts": {
+    "precommit": "npm run test",
```

Comment:
Discussed with @jamo and we agreed that we should have `prepush` running XO only (we'd need to introduce a `lint` task).
@leo 👌 ❓